cnf-certification-test/platform/cnffsdiff/fsdiff.go

```diff
@@ -41,26 +39,20 @@ const (
 
 type FsDiff struct {
 	containerID string
-	command     []string
+	command     string
 	result      int
 }
 
 func NewFsDiff(c *provider.Container) (*FsDiff, error) {
-	id := c.Status.ContainerID
-	split := strings.Split(id, "://")
-	uid := ""
-	if len(split) > 0 {
-		uid = split[len(split)-1]
-	}
-	if uid == "" {
-		logrus.Debugln(fmt.Sprintf("could not find uid of %s/%s/%s\n", c.Namespace, c.Podname, c.Data.Name))
-		return nil, errors.New("CAN'T INITIALIZE FSDIFF INSTANCE")
+	uid, err := c.GetUID()
```

Comment:
added the uid/runtime in the config as part of porting the networking tests PR. We can merge it in another PR. https://github.com/test-network-function/cnf-certification-test/pull/33

Reply:
Sounds good.  Let's get this one merged.